Hcal/src/Hcal/HcalDoubleEndRecProducer.cxx

```diff
@@ -105,21 +105,21 @@ void HcalDoubleEndRecProducer::produce(framework::Event& event) {
 
     int position_bar_sign = hitTimeDiff > 0 ? 1 : -1;
     double position_unchanged = 0;
-    int isX = 0;
+    int bar_orientation = 0;
     double position_bar = position_bar_sign * fabs(hitTimeDiff) * v / 2;
     if (orientation ==
         ldmx::HcalGeometry::ScintillatorOrientation::horizontal) {
       position_unchanged = position.X();
-      isX = 1;
+      bar_orientation = 1;
       position.SetX(position_bar);
     } else {
       position_unchanged = position.Y();
-      isX = 0;
+      bar_orientation = 0;
```

Comment:
Again i would just persist the enumeration directly. It should be written to disk as an integer anyways